**Задача 3. На соревновании по биатлону один из трех спортсменов стреляет и попадает в мишень. Вероятность попадания для первого спортсмена равна 0.9, для второго — 0.8, для третьего — 0.6. Найти вероятность того, что выстрел произведен:** 
- **a)первым спортсменом**
- **б)вторым спортсменом**
- **в)третьим спортсменом**

**Решение:**\
По условию задачи понятно, необходимо определить вероятность события при условии, что другое событие уже закончилось (выстрел в мишень уже произведем). Для нахождения вероятности используем формулу Байеса:\
**P(A/B) = P(B/A) * P(A) / P(B)**, где\
событие **A** уже завершилось, выстрел сделан и сделал его один из трех спортсменов, поэтому **P<sub>1</sub> + P<sub>2</sub> + P<sub>3</sub> = 1/3**\
P(A/B) - вероятность, что какой-либо спортсмен произвел выстрел\
P(B/A) - вероятность попадания каждого спортсмена\
P(B) - полная вероятность события A - вычисляется по формуле:\
**P(B) - P(B/A<sub>1</sub>) * P(A<sub>1</sub>) + P(B/A<sub>2</sub>) * P(A<sub>2</sub>) + P(B/A<sub>2</sub>) * P(A<sub>2</sub>)**

In [12]:
PBa1 = 0.9
PBa2 = 0.8
PBa3 = 0.6
PB = 0.9 * 1/3 + 0.8 * 1/3 + 0.6 * 1/3
print(f'Полная вероятность наступления события A равна: {PA: .4f}')

Полная вероятность наступления события A равна:  0.7667


a) Вычислим вероятность, что выстрел произвел первый спортсмен:

In [13]:
p1 = PBa1 * 1/3 / PB
print(f'Вероятность выстрела первого спортсмена: {p1: .4f}')

Вероятность выстрела первого спортсмена:  0.3913


б) Вычислим вероятность, что выстрел произвел второй спортсмен:

In [14]:
p2 = PBa2 * 1/3 / PB
print(f'Вероятность выстрела первого спортсмена: {p2: .4f}')

Вероятность выстрела первого спортсмена:  0.3478


в) Вычислим вероятность, что выстрел произвел третий спортсмен:

In [15]:
p3 = PBa3 * 1/3 / PB
print(f'Вероятность выстрела первого спортсмена: {p3: .4f}')

Вероятность выстрела первого спортсмена:  0.2609


Ответ:\
    Вероятность того, что выстрел произвел:\
        - первый спортсмен = 39% \
        - второй спортсмен = 35% \
        - третий спортсмен = 26%